python/tests/metta.py

```diff
@@ -1,10 +1,13 @@
 import os
 import sys
+from hyperonpy import init_logger
+
 
 from common import MeTTa
 
 if __name__ == "__main__":
     os.system('clear')
+    init_logger()
```

Comment:
This is not needed anymore after #80 is merged.

Reply:
It's needed on my machine. I don't have logs without this line.

Reply:
It is strange, could you please check if https://github.com/trueagi-io/hyperon-experimental/blob/067d627e708e9841f1cda9dcc2df479d809642eb/lib/src/lib.rs#L14 is called when `hyperon` library is loaded from Python in your environment?
For example you can run Python in GDB like this:
```
$ gdb --args python -c 'import hyperon'
(gdb) break on_load
(gdb) r
```

Reply:
It stopped working starting from `rustc` version `1.60`.

Reply:
But it will be fixed in 1.63. You can roll back rust toolchain to `1.59` to make it work

Reply:
Fixed it in #108